vignettes/pkgdown.Rmd

```diff
@@ -105,18 +105,20 @@ reference:
 
 Note the use of `starts_with()` to select all functions with a common prefix.
 You can also use `ends_with()` and `matches()`.
-See complete details in `?build_reference`.
+See complete details in `?build_reference`, including other topic matching helper functions.
 
 While iterating on the reference index you might want to run `pkgdown::build_reference_index()`.
 It just re-builds the index page, making it faster to quickly change `_pkgdown.yml` and see how it affects your site.
 
 ## Articles
 
 pkgdown will automatically build all vignettes found in `vignettes/`, translating them to HTML files in `articles/`.
-Due to the way that pkgdown has to integrate RMarkdown generated HTML with its own HTML, relatively little control is available over the output format.
+It is recommended to name your intro article with your package name to generate a Get Started page automatically.
+
+Due to the way that pkgdown has to integrate R Markdown generated HTML with its own HTML, relatively little control is available over the output format.
 You can see the details in `?build_articles`.
 
-If you want to include an article on the website but not in the package (e.g., because it's large), you can either place it in a subdirectory of `vignettes/` (e.g. `vignettes/web_only`) or add it to `.Rbuildignore` (and make sure that there's no `vignettes:` section in the yaml header).
+If you want to include an article on the website but not in the package (e.g., because it's large), you can either place it in a subdirectory of `vignettes/` (e.g. `vignettes/web_only`) or add it to `.Rbuildignore` (and make sure that there's no `vignettes:` section in the yaml header). See `usethis::use_article()` for an automated way to do this.
```

Comment:
```suggestion
If you want to include an article on the website but not in the package (e.g., because it's large), you can either place it in a subdirectory of `vignettes/` (e.g. `vignettes/articles`) or add it to `.Rbuildignore` (and make sure that there's no `vignettes:` section in the yaml header). See `usethis::use_article()` for an automated way to do this.
```

Do we really need to suggest two ways to do this?